test/test_contest.c

```diff
@@ -0,0 +1,56 @@
+#include "test.h"
+
+#include "../src/dxcc.h"
+#include "../src/globalvars.h"
+#include "../src/setcontest.h"
+
+
+
+// OBJECT ../src/dxcc.o
+// OBJECT ../src/setcontest.o
+
+/* dummys */
+int getctynr() {
+    return 42;
+}
+
+contest_config_t config_focm;
+
+int setup_default(void **state) {
+    char filename[100];
+
+    strcpy(filename, TOP_SRCDIR);
+    strcat(filename, "/share/cty.dat");
```

Comment:
could be written as
```
static char filename[] = TOP_SRCDIR "/share/cty.dat"
```
(to avoid eventual buffer overflow)

Reply:
Good idea. See new commit.